### PR TITLE
linijki 9-18

### PR DESCRIPTION
**Pull request by meVeri**
dodałem parę poleceń, trochę skróciłem by jednak strona odpalała. Udało się do tą uzyskać efekt że o imię pyta xD

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -2,9 +2,18 @@ import Head from "next/head";
 import Image from "next/image";
 import { Inter } from "next/font/google";
 import styles from "@/styles/Home.module.css";
+import { Console } from "console";
```
Comment (theXaMi) [inline pages/index.tsx:5]: That's unnecessary. It literally does nothing.

File: pages/index.tsx [hunk 2]
```diff
@@ -2,9 +2,18 @@ import Head from "next/head";
...
 
 const inter = Inter({ subsets: ["latin"] });
 
+const name = "Whats your name";
```
Comment (theXaMi) [inline pages/index.tsx:9]: I hope you realize that this doesn't magically get the name of the user 😛

**Review by theXaMi — CHANGES_REQUESTED** [review]
A bit too much but ok. I'd remove the unnecessary import as it may give you bad ideas, other than that it's ok.

**Comment by meVeri** [comment]
Well, I was kind of hoping that this would allow me to get the "name" and
display it in the greeting 😅

sob., 26 sie 2023 o 18:34 Michał Modzelewski ***@***.***>
napisał(a):

**Pull request merged**